static/app/bootstrap/renderSystemAlerts.tsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react';
+import ReactDOM from 'react-dom';
+
+import SystemAlerts from 'app/views/app/systemAlerts';
+
+export function renderSystemAlerts(container: string, props?: Record<string, any>) {
```

Comment:
Opted to not abstract these as they are simple enough, and hopefully will not need to add any more.